apps/ledger-live-desktop/src/renderer/screens/exchange/Swap2/Form/index.tsx

```diff
@@ -98,29 +77,29 @@ export const useProviders = () => {
     providersError,
   };
 };
+
 const SwapForm = () => {
-  // FIXME: should use enums for Flow and Banner values
-  const [currentFlow, setCurrentFlow] = useState<string | null>(null);
-  const [currentBanner, setCurrentBanner] = useState<string | null>(null);
   const swapDefaultTrack = useGetSwapTrackingProperties();
   const [idleState, setIdleState] = useState(false);
-  const [error, setError] = useState<ValidCheckQuoteErrorCodes | undefined>();
+  const [error, setError] = useState<ValidCheckQuoteErrorCodes | undefined>(); // TODO: delete?
```

Comment:
It seems to only be linked to the `checkQuote` logic that was used for FTX. Could probably be cleaned up / removed.

cf.
- https://github.com/LedgerHQ/ledger-live/blob/support/live-8503-remove-ftx-wyre-code-lld/libs/ledger-live-common/src/exchange/swap/types.ts#L104-L133
- https://github.com/LedgerHQ/ledger-live/blob/support/live-8503-remove-ftx-wyre-code-lld/libs/ledger-live-common/src/exchange/swap/checkQuote.ts

Also, you should get rid of all the KYC related logic in LLC as well
cf.
- https://github.com/LedgerHQ/ledger-live/blob/support/live-8503-remove-ftx-wyre-code-lld/libs/ledger-live-common/src/exchange/swap/types.ts#L8-L24

One possibility would be to start by removing the types (cf. links above) and pull up the thread to the actual functions 🧹